django_auth_ldap/backend.py

```diff
@@ -793,7 +778,8 @@ def _mirror_groups(self):
             new_groups = [Group.objects.get_or_create(name=name)[0] for name
                           in target_group_names if name not in existing_group_names]
 
-            self._user.groups = existing_groups + new_groups
+            # self._user.groups = existing_groups + new_groups
```

Comment:
do we want to keep this commented-out line around?

Reply:
No, I was supposed to clean that up